### PR TITLE
Fix delete warning message #1051 

### PR DESCRIPTION
**Pull request by groovecoder**
Supersedes https://github.com/mozilla/multi-account-containers/pull/1205 with updated `package.json` `devDependencies` to fix Travis.

### CODE REVIEW

**Comment by groovecoder** [comment]
@ShivangiKakkar can you give a quick review/r+ on this? Functionality still works and the code still seems good.

File: src/js/popup.js [hunk 1]
```diff
@@ -1074,9 +1078,20 @@ Logic.registerPanel(P_CONTAINER_DELETE, {
   prepare() {
     const identity = Logic.currentIdentity();
 
-    // Populating the panel: name and icon
+    // Populating the panel: name, icon, and warning message
     document.getElementById("delete-container-name").textContent = identity.name;
 
+    const totalNumberOfTabs = identity.numberOfHiddenTabs + identity.numberOfOpenTabs;
```
Comment (groovecoder) [inline src/js/popup.js]: Whoops, I approved too early. An optional refinement to eliminate a little bit of duplicate functionality ...
```
     let warningMessage = "";
     if (totalNumberOfTabs > 0) {
      const grammaticalNumTabs = totalNumberOfTabs > 1 ? "tabs" : "tab";
      warningMessage = `If you remove this container now, ${totalNumberOfTabs} container ${grammaticalNumTabs} will be closed.`;
    }
    document.getElementById("delete-container-tab-warning").textContent = warningMessage;
```

Comment (ShivangiKakkar) [reply src/js/popup.js]: @groovecoder Hey! sorry for the delay, this surely makes the logic concise. Should I make these changes and push?

Comment (groovecoder) [reply src/js/popup.js]: Yes pelase. 👍

Comment (ShivangiKakkar) [reply src/js/popup.js]: Pushed the changes, but the Travis build seems to be failing somehow 😭

**Pull request merged**